packages/jsii-pacmak/lib/targets/golang.ts

```diff
@@ -30,8 +31,11 @@ export class Golang extends Target {
 class GolangGenerator implements IGenerator {
   private assembly!: Assembly;
   private readonly code = new CodeMaker();
+  private readonly documenter: Documentation;
 
-  public constructor(private readonly rosetta: Rosetta) {}
+  public constructor(private readonly rosetta: Rosetta) {
+    this.documenter = new Documentation(this.code, this.rosetta);
```

Comment:
I don't really like passing in the CodeMaker in this way, but I wanted the documenter object to encapsulate the doc generation logic.